libs/application/templates/party-letter/src/fields/index.ts

```diff
@@ -0,0 +1 @@
+export {}
```

Comment:
Is that working?

Reply:
Yes wanted to keep the structure since we expect to use custom fields later on.
We are returning an empty module to prevent errors